### DISCUSSION TITLE
Berbat Türkçe hk

### DISCUSSION THREAD

**Discussion by ghost** [VeriTeknik/linux_kitap#3] [General]
Dökümantasyonu yazan arkadaşın teknik bilgi ve donanıma sahip olduğunu düşünmekle birlikte berbat bir Türkçesinin olduğunu söylemeden geçemeyeceğim. Dökümantasyonlarda sıklıkla karşıma çıkan yazım yanlışlarının bir listesini aşağıya bırakıyorum. 

- `bir kaç`  ❌
- Doğrusu: birkaç : ✅

- `bir çok`  ❌
- Doğrusu: birçok : ✅

- `pekçok`  ❌
- Doğrusu: pek çok : ✅

Ayrıca yüklemden sonra virgül gelmez. Dökümantasyonlarınızda türkçe cinayeti işleniyor. Bir yerden sonra sinirlenerek okumayı bıraktım. Üzgünüm.

**Comment by ckaraca** [comment]
Uyarınız için teşekkür ederiz, sonraki revizyonlarda düzeltmeleri yapacağız.
